07_errors/n0npax/store_test.go

```diff
@@ -0,0 +1,199 @@
+package main
+
+import (
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type storerImpl int
+
+const (
+	syncStorer storerImpl = iota
+	memStorer
```

Comment:
I'll stand by my [review comments from lab 10](https://github.com/anz-bank/go-course/pull/504#discussion_r308493171), please change or convince me otherwise.

Reply:
I hope it's fixed In `lab10` and `lab07`